org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java

```diff
@@ -778,67 +774,70 @@ private void addVariableDeclarations(DashSuperState state) {
                 }
             }
         }
-        for(Decl decl: state.getVariables()) {
 
-            String stateName = state.getFullyQualName();
-            String variableName = decl.get().toString();
-            variable2StateNameMap.put(variableName, stateName);
+        // add the state variables
+        for(Decl decl: state.getVariables()) {
+            for (ExprHasName var : decl.names) {
+                String stateName = state.getFullyQualName();
+                String variableName = var.toString();
```

Comment:
Previously, we only gather the first variable in the declaration. However, there could be multiple.